src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import './navbar.css';
+import ScrollTop from '../ScrollTop.jsx';
+
+const Navbar = () => {
+    const [scrolled, setScrolled] = useState(false);
+    const [burgerOpen, setBurgerOpen] = useState(false);
+    const location = useLocation();
+
+    useEffect(() => {
+        const handleScroll = () => {
+            const offset = window.scrollY;
+            if(offset > 50){
+                setScrolled(true);
+            }
+            else{
+                setScrolled(false);
+            }
+        };
+        window.addEventListener('scroll', handleScroll);
+
+        return () => {
+            window.removeEventListener('scroll', handleScroll);
+        }
+    }, []);
+
+    let navbar = 'navbar';
+    if(scrolled){
+        navbar += ' navbar__scrolled';
+    }
+
+    const burgerClick = () => {
+        setBurgerOpen(!burgerOpen);
+    }
+
+    const isActive = (path) => location.pathname === path ? 'active' : '';
+
+    return (
+        <div className={navbar}>
+            <ScrollTop></ScrollTop>
+            {/* logo */}
+            <Link to="/">
+                <img src={process.env.PUBLIC_URL + '/assets/jollybeelogo.png'} alt="JB Logo" className='logo'></img>
+            </Link>
```

Comment:
Currently, the `a` tag for navbar logo spans too big because it is affected by flex property.

<details>
<summary>Image</summary>

![image](https://github.com/user-attachments/assets/8044f9bd-eacb-43a8-baab-b38dea2754aa)
</details>

Add a `div` tag so that the `a` tag is not affected by flex property.

```suggestion
            <div>
                <Link to="/">
                    <img src={process.env.PUBLIC_URL + '/assets/jollybeelogo.png'} alt="JB Logo" className='logo'></img>
                </Link>
            <div>
```